### ISSUE TITLE
[FR] Route output audio to microphone input.

### ISSUE THREAD

**Issue by reesericci** [wwmm/easyeffects#1420]
It would be really cool if you could provide a simple way to route my output speaker to my input sink. That way, I don't have to do it in an error-prone way with Helvum per output source. Basically, I want to create a knock-off Loopback: https://rogueamoeba.com/loopback/.

**Comment by wwmm** [comment]
I am not sure this is a good fit for EasyEffects. Managing links between devices in a generic way is the Helvum domain. WE already have a lot of complex scenarios to deal with. Maybe improving Helvum in a way this task is less error prone is the best thing to do.

**Comment by reesericci** [comment]
Yeah, but I think that EasyEffects would provide a significantly more usable interface as compared to a full-on patchbay. I just want to route my speakers to my mic and add some effects, y'know?

**Comment by wwmm** [comment]
Let's keep this opened for some time. There are some features like multiple filters instances that are at the top of our priority list but maybe in the future we can look at this.

**Comment by reesericci** [comment]
OK, no problem. Those features sound awesome!

**Comment by wwmm** [comment]
EasyEffects 8.x is capable of sending the audio processed by the output effects pipeline to the our virtual source. It should be enough.

**Comment by UltraBlackLinux** [comment]
how does that work? I can't find an option to enable that...
Edit: Button is in the bottom right of the output section

**Comment by wwmm** [comment]
>Edit: Button is in the bottom right of the output section

Maybe I should add text to this button instead of just putting an icon. But I am not sure about the text. Ideally it should be something short.

**Comment by UltraBlackLinux** [comment]
I think a toggle switch for input monitoring and sending audio to the virtual source in the settings would be completely fine. That way you could add a text label without adding clutter to the UI. Maybe even remove the buttons entirely. The button to send audio to the input isn't really something you need quick access to in my opinion.

But maybe change the icon of the send audio to input button - the share indicator kind of has its meaning already pre-determined so it's more confusing here than anything else.
